### PR TITLE
ci: firedancer integration test

### PR DESCRIPTION
**Pull request by lheeger-jump**

### CODE REVIEW

File: .github/workflows/test_firedancer.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+        with:
+          count: 128
+
+      - name: build
+        run: |
+          ./contrib/make-j fddev all
+      - name: test firedancer - local cluster
+        run: |
+          sudo prlimit --pid=$$ --nofile=1048576
+          sudo prlimit --pid=$$ --memlock=unlimited
```
Comment (mmcgee-jump) [inline .github/workflows/test_firedancer.yml:28]: You don't need these, fddev handles nofile and memlock.

Comment (lheeger-jump) [inline .github/workflows/test_firedancer.yml:28]:
```suggestion
```

Comment (lheeger-jump) [reply .github/workflows/test_firedancer.yml]: done

**Review by arjain4 — APPROVED** [review]

File: .github/workflows/test_firedancer.yml (hunk 1 above)
Comment (lheeger-jump) [reply .github/workflows/test_firedancer.yml:28]: nvm, undid. these are set up for a local cluster we set up for the script. fddev doesnt do this until much later.

**Review by arjain4 — APPROVED** [review]

**Pull request merged**